wandb/env.py

```diff
@@ -363,6 +365,21 @@ def get_cache_dir(env: Optional[Env] = None) -> str:
     return val
 
 
+def get_minimum_free_space(
+    env: Optional[Env] = None, default: int = 1_000_000_000
```

Comment:
should we default to 0 to maintain backcompat since effectively the min is 0 today. Otherwise, I'm worried this change will cause errors for embedded systems or something and require the user to set a new variable

Reply:
Possible. Or some small value, probably 1MB is still fairly safe in terms of not borking file systems while still being expected even in pretty resource constrained environments. I think it should be at minimum 64k though.

Reply:
a small minimum sounds fine, if you dont have 64k of 1mb of disk, you probably shouldn't be using W&B, i think our sdk will just blow up your filesystem :P